src/base/dof_map_constraints.C

```diff
@@ -702,7 +702,7 @@ private:
               for (unsigned int qp=0; qp<n_qp; qp++)
                 {
                   // solution at the quadrature point
-                  OutputNumber fineval = 0;
+                  OutputNumber fineval(0);
```

Comment:
No; I had the same errors.  I was using gcc 4.8.0, though, so there may be something compiler-specific going on.